.github/workflows/chatbot.yaml

```diff
@@ -74,3 +85,48 @@ jobs:
           image: ${{ steps.build_image.outputs.image }}
           tags: ${{ steps.build_image.outputs.tags }}
           registry: ${{ env.REGISTRY }}
+
+      - name: Get changed chatbot files
```

Comment:
What if the workflow file changes? Or, could there be other files on top of the chatbot folder?

Reply:
eventually, we'll gate running this with the `paths:` field - it does the same as this changed_files step, however, it will gate _all_ steps on this - with this, I wanted to only run the extra build & push on merge events. - I'll add to the changed files `.github/workflows/chatbot.yaml`

Reply:
@lmilbaum I've moved the quay.io image build/pushes to the testing-framework workflow, ptal